src/test/java/com/cloudbees/plugins/credentials/CredentialsParameterDefinitionTest.java

```diff
@@ -141,7 +146,7 @@ public void escapeAndMarkupFormatAreDoneCorrectly() throws Exception {
         HtmlPage page = wc.getPage(p, "build?delay=0sec");
         collector.checkThat(page.getWebResponse().getStatusCode(), is(HttpURLConnection.HTTP_BAD_METHOD)); // 405 to dissuade scripts from thinking this triggered the build
         String text = page.getWebResponse().getContentAsString();
-        collector.checkThat("build page should escape param name", text, containsString("&lt;param name&gt;"));
+        collector.checkThat("build page should escape param name", text, Matchers.containsString("&lt;param name&gt;"));
```

Comment:
containsString is already imported (albeit from `CoreMatchers` but they are the same).

```suggestion
        collector.checkThat("build page should escape param name", text, containsString("&lt;param name&gt;"));
```